builder/llpcBuilderImplImage.cpp

```diff
@@ -864,6 +864,34 @@ static const Intrinsic::ID ImageAtomicIntrinsicTable[][8] =
     },
 };
 
+// =====================================================================================================================
+// Convert an integer or vector of integer type to the equivalent (vector of) half/float/double
+static Type* ConvertTypeToFp(
+    Type* pOrigTy)    // Original type
+{
+    LLPC_ASSERT(pOrigTy->isIntOrIntVectorTy());
+    Type* pNewTy = pOrigTy;
+    switch (pNewTy->getScalarType()->getIntegerBitWidth())
+    {
+    case 16:
+        pNewTy = Type::getHalfTy(pNewTy->getContext());
+        break;
+    case 32:
+        pNewTy = Type::getFloatTy(pNewTy->getContext());
+        break;
+    case 64:
```

Comment:
I don't think our HW support 64bit type? have you hit this case in CTS or some other test?
if not, please add LLPC_NEVER_CALLED()  before your code.